.github/workflows/clean-style-lint.yml

```diff
@@ -22,20 +27,37 @@ jobs:
           make clean
           make clean_tt
           make style
-      - name: Import GPG key
+      
+      #!TODO: remove in future -- after a few runs 
```

Comment:
Do you need this block of commented out code? What's the intent of keeping it in?

Reply:
I don't need it ; forgot to remove it :)